roles/atomic_host_check/tasks/main.yml

```diff
@@ -6,21 +6,9 @@
 # Logic borrowed from:
 # https://github.com/kubernetes/contrib/blob/master/ansible/roles/common/tasks/main.yml
 #
-- name: Determine if Atomic Host
-  stat:
-    path: "/run/ostree-booted"
-  register: s
-  changed_when: false
-  check_mode: no
 
-- name: Init the is_atomic fact
-  set_fact:
-    is_atomic: false
-
-- name: Set the is_atomic fact
-  set_fact:
-    is_atomic: true
-  when: s.stat.exists
+- name: Determine is_atomic True/False
+  include: '{{ playbook_dir }}/common/set_is_atomic.yml'
```

Comment:
Hooray for CI!  Our `redhat-ci` checks found this problem:

```
TASK [atomic_host_check : Determine is_atomic True/False] **********************
fatal: [testnode]: FAILED! => {
    "failed": true, 
    "reason": "the file_name '/var/tmp/checkout/tests/improved-sanity-test/common/set_is_atomic.yml' does not exist, or is not readable"
}
```

Looks like the `include:` statement is pointing at the wrong place.